From c443a1c5d64d4827e12bd914b3925150fc202692 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=89lie=20Bouttier?= <elie@bouttier.eu>
Date: Mon, 31 Oct 2016 21:06:32 +0100
Subject: [PATCH] Remove ExecStop in tinc@.service

This avoid tinc to receive SIGTERM twice (through ExecStop and through systemd
directly) which prevented tinc-down script to be executed.
---
 distro/tinc@.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/distro/tinc@.service b/distro/tinc@.service
index 4bb4d2dd..c823fa5a 100644
--- a/distro/tinc@.service
+++ b/distro/tinc@.service
@@ -8,7 +8,6 @@ Type=simple
 WorkingDirectory=/etc/tinc/%i
 ExecStart=/usr/sbin/tincd -n %i -D
 ExecReload=/usr/sbin/tincd -n %i -kHUP
-ExecStop=/usr/sbin/tincd -n %i -k
 TimeoutStopSec=5
 Restart=always
 RestartSec=60
-- 
2.10.2

